src/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -378,6 +378,7 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
         BN254.G1Point memory pubkey, 
         string memory socket
     ) internal virtual returns(uint32[] memory) {
+        // TODO after slashing: revert registration when operator is slashable in the EigenLayer Slasher
```

Comment:
nit: this should be when they are *not* slashable (but the interface might change so... 🤷 )

Reply:
I'm fine with that 👍

Reply:
Wait interface might change but we will always require being opted into slashing to be able to register right?
Feels like we should remove the commented code below (since the interface will change), but not this comment?

Reply:
> Wait interface might change but we will always require being opted into slashing to be able to register right? Feels like we should remove the commented code below (since the interface will change), but not this comment?

tbh I'm fine either way. The communication about what being "opted into slashing" means might change, but having a TODO left in the code seems fine